unittests/ASM/CMakeLists.txt

```diff
@@ -43,21 +43,28 @@ foreach(ASM_SRC ${ASM_SOURCES})
 
   # Format is "<Test Arguments>" "<Test Name>"
   set(TEST_ARGS
-    "-c irint -n 1"      "int_1"
-    "-c irint -n 500"    "int_500"
-    "-c irint -n 500 -m" "int_500_m"
-    "-c irjit -n 1"      "jit_1"
-    "-c irjit -n 500"    "jit_500"
-    "-c irjit -n 500 -m" "jit_500_m"
+    "-c irint -n 1"      "int_1"     "int"
+    "-c irint -n 500"    "int_500"   "int"
+    "-c irint -n 500 -m" "int_500_m" "int"
+    "-c irjit -n 1"      "jit_1"     "jit"
+    "-c irjit -n 500"    "jit_500"   "jit"
+    "-c irjit -n 500 -m" "jit_500_m" "jit"
     )
+  if (_M_X86_64)
+    list(APPEND TEST_ARGS
+      "-g -c host"       "host"      "host"
```

Comment:
why not pass -g globally? seems like a good idea anyway

Reply:
Sounds good